images/scripts/arch.setup

```diff
@@ -29,6 +29,8 @@ packagekit \
 libvirt \
 libvirt-python \
 libvirt-dbus \
+libpwquality \
+sssd \
```

Comment:
Is this really right? cockpit shouldn't depend on sssd, that should be an optional add-on. I.e. it may make more sense as a test dependency? We use `$IPA_CLIENT_PACKAGES` for that in the other image scripts.

Of course it does not make a difference in the built image, but it's nice to keep cockpit's own deps and "things that it can talk to" separate.

Reply:
Ok, I will move it there.